File: Random/JAVA/DSA/OddEven.java
public class OddEven{



    public static void main(String[] args) {
        int[] nums = { 1,53,3,45,12,53,12,3,53,2};

        for( int i = 0 ; i < nums.length ; i++){
            if((nums[i] & 1 )== 1){
                System.out.println("Odd Number : " + nums[i]);

            }
            else{
                System.out.println("Number is Even : " + nums[i]);
            }
        }

    }



}

File: Random/JAVA/DSA/OddEven.py
class OddEven : 
    
    
    def getTheNumber(self, nums): 
        for i in nums:
            if((i & 1) == 1 ):
                print("Number is Odd : ", i)

            else : 
                print("Number is Even : ", i)
                
nums = [1, 4,3,6,3,63,74,534,63,42,32]
sol = OddEven()
sol.getTheNumber(nums)
                
        